truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -347,10 +349,13 @@ def copy_into_build_dir(from_path: Path, path_in_build_dir: str):
                 )
 
             config.base_image = BaseImage(
-                image=TRTLLM_BASE_IMAGE, python_executable_path="/usr/bin/python3"
+                image=TRTLLM_BASE_IMAGE_AND_EXECUTABLE[0],
+                python_executable_path=TRTLLM_BASE_IMAGE_AND_EXECUTABLE[1],
             )
             config.requirements.extend(BASE_TRTLLM_REQUIREMENTS)
 
+            config.model_metadata["tags"] = ["openai-compatible"]
```

Comment:
nit: We have this `"openai-compatible"` string both here and in the model template. Can we move it to the `constants.py` and reuse it?